cake-recipe.txt

```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup canola oil
```

Comment:
its better if we can use any healthy oil like, mustard, olive, seasame, sunflower etc, Let me know whe you have updated the change